src/index.js

```diff
@@ -263,7 +263,7 @@ class Dropzone extends React.Component {
   onInputElementClick(evt) {
     evt.stopPropagation()
     if (this.props.inputProps && this.props.inputProps.onClick) {
-      this.props.inputProps.onClick()
+      this.props.inputProps.onClick(evt)
```

Comment:
There is a test case (`should invoke inputProps onClick if provided`) which checks the `onClick()` behaviour, could you update the test to reflect your changes (cb is called with evt)?

Reply:
definitely, sorry I missed it on the first pass.

Reply:
@rolandjitsu test is now updated.